interface/patient_file/summary/pnotes_full.php

```diff
@@ -131,15 +131,16 @@
     } elseif ($mode == "new") {
         $note = $_POST['note'];
         if ($noteid) {
-            updatePnote($noteid, $note, $_POST['form_note_type'], $_POST['assigned_to']);
+            updatePnote($noteid, $note, $_POST['form_note_type'], $_POST['assigned_to'], '', !empty($_POST['form_datetime']) ? $_POST['form_datetime'] : '');
```

Comment:
for example, this is where should do the DateTimeToYYYYMMDDHHMMSS() conversion

Reply:
Done